addons/spreadsheet/static/src/pivot/pivot_time_adapters.js

```diff
@@ -54,10 +54,21 @@ const odooNumberDateAdapter = {
         return Number(readGroupResult[groupBy]);
     },
     increment(normalizedValue, step) {
-        return normalizedValue + step;
+        return normalizedValue + step
```

Comment:
The lien was changed to delete the `;`  ? :sweat_smile:

Reply:
Oops :sweat_smile: